client.go

```diff
@@ -706,12 +710,12 @@ func (c *Client) Server(ctx context.Context) (ServerInfo, error) {
 	}
 	c.connMu.Unlock()
 
-	response, err := c.IntrospectionService().Server(ctx, &types.Empty{})
+	uuid, err := c.IntrospectionService().Server(ctx, &types.Empty{})
 	if err != nil {
 		return ServerInfo{}, err
 	}
 	return ServerInfo{
-		UUID: response.UUID,
+		UUID: uuid,
```

Comment:
Any specific reason for this change?

Reply:
This was just to match the new returned values.

Reply:
I'm going to make this change back to be more consistent.